src/main/kotlin/no/nav/syfo/melding/kafka/legeerklaring/LegeerklaringVedleggDTO.kt

```diff
@@ -0,0 +1,21 @@
+package no.nav.syfo.melding.kafka.legeerklaring
+
+import java.util.Base64
+
+data class LegeerklaringVedleggDTO(
+    val vedlegg: Vedlegg,
+)
+
+fun LegeerklaringVedleggDTO.getBytes() =
+    Base64.getMimeDecoder().decode(vedlegg.content.content)
```

Comment:
Er det en grunn til at denne funksjonen er en extension function og ikke en del av `data class`'en? Tenker at siden funksjonaliteten ligger så nært propertien til klassen at den burde være en del av klassen. Hva tenker du? 😊
```suggestion
data class LegeerklaringVedleggDTO(
    val vedlegg: Vedlegg,    
) {
    fun getBytes() =
        Base64.getMimeDecoder().decode(vedlegg.content.content)
}
```

Reply:
Tenker det er greit med extension function når det er snakk om en dto-klasse som skal matche det vi får fra en ekstern app :+1: